project/Dependencies.scala

```diff
@@ -25,6 +26,7 @@ object Dependencies {
     val mockitoScala = "org.mockito" %% "mockito-scala-scalatest" % versions.mockitoScala % Test
   }
 
+  val testInterface = "org.scala-sbt" % "test-interface" % versions.testInterface
```

Comment:
Curious. What happens if the project under test depends on a different version of this interface?

Reply:
The last version of this package was published in 2013, so I would say that's a pretty stable API. Even so, I've now changed the dependency to 'Provided' (so assumed to be there). As long as there are no breaking changes used in the API it should be fine